src/main/java/com/terraformersmc/cinderscapes/init/CinderscapesPlacementModifierTypes.java

```diff
@@ -0,0 +1,20 @@
+package com.terraformersmc.cinderscapes.init;
+
+
+import com.terraformersmc.cinderscapes.placementModifier.SafelistRangeCeilingPlacementModifier;
+import com.terraformersmc.cinderscapes.placementModifier.SafelistRangeFloorPlacementModifier;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.world.gen.placementmodifier.PlacementModifierType;
+
+
+// TODO: Check
+public class CinderscapesPlacementModifierTypes {
+    public static PlacementModifierType<SafelistRangeCeilingPlacementModifier> COUNT_CEILING;
+    public static PlacementModifierType<SafelistRangeFloorPlacementModifier> COUNT_FLOOR;
```

Comment:
Use the `final` modifier and initialize the placement modifier types here rather than during registration.